integration-tests/rpc-path-root/test/index.test.js

```diff
@@ -49,11 +49,16 @@ function runTests(dev = false) {
     it(
       "monorepo query works",
       async () => {
-        const data = await fetchViaHTTP(appPort, "/api/rpc/queries/getNoSuspenseBasic", null, {
-          method: "POST",
-          headers: {"Content-Type": "application/json; charset=utf-8"},
-          body: JSON.stringify({params: {}}),
-        }).then((res) => res.ok && res.json())
+        const data = await fetchViaHTTP(
+          appPort,
+          "/api/rpc/no-suspense/app/queries/getNoSuspenseBasic",
+          null,
+          {
+            method: "POST",
+            headers: {"Content-Type": "application/json; charset=utf-8"},
+            body: JSON.stringify({params: {}}),
+          },
+        ).then((res) => res.ok && res.json())
```

Comment:
this test should not change. For this app with it's current config, the rpc path should be `/api/rpc/queries/getNoSuspenseBasic`.

so we need to app a **new** integration test to reproduce and test for the issue that @paulm17 is having